xikolo-ios/RichtextViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  RichtextViewController.swift
+//  xikolo-ios
+//
+//  Created by Bjarne Sievers on 17.06.16.
+//  Copyright © 2016 HPI. All rights reserved.
+//
+
+import UIKit
+
+class RichtextViewController: AbstractItemRichtextViewController {
+
+    @IBOutlet weak var titleLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        titleLabel.text = courseItem?.title
```

Comment:
You have defined `courseItem` as `CourseItem!`, an implicitly unwrapped optional. If there's a need to check for `nil` here (using `?`), it should be a real optional. If there isn't, remove the `?`.